File: stress/t/flt_basel.f
      PROGRAM FBASEL
      IMPLICIT DOUBLE PRECISION (A-H,O-Z)
      INTEGER*8 K,N
      N=5000
      SUM=0.0D0
      DO 10 K=1,N-1
        AKF=DBLE(K)
        D=AKF*AKF
        SUM=SUM+1.0D0/D
   10 CONTINUE
      WRITE(*,'(A,F0.6)') 'sum= ',SUM
      END

File: stress/t/flt_basel.c
#include <stdio.h>
#include <math.h>
int main(void){
  long k,n=5000; double sum=0.0,d,kf;
  for(k=1;k<n;k++){ kf=(double)k; d=kf*kf; sum=sum+1.0/d; }
  printf("sum= %f\n",sum);
  return 0;
}
